.changeset/mean-ads-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+"comet-api-v10": major
```

Comment:
why is this a major change? shouldnt this be a minor?

Reply:
Version was changed.

af3a715725c25596b1bf4b81cf29f121ee9e00ea

Reply:
Changed doc. for changeset

e028339143d01cc01318ab8f84d216c1c8f4cfc8